src/test/kotlin/com/bardsoftware/papeeria/backend/cosmas/CosmasGoogleCloudServiceTest.kt

```diff
@@ -74,11 +88,52 @@ class CosmasGoogleCloudServiceTest {
         assertNotNull(stream2.error)
     }
 
-    private fun getFileFromService(version: Int, fileId: String = "0", projectId: String = "0"): ByteString {
+    @Test
+    fun addSecondVersionAndCheckListVersions() {
+        val fakeStorage: Storage = mock(Storage::class.java)
+        val blob1 = getMockedBlob("ver1".toByteArray(), 1)
```

Comment:
I suggest passing string and converting toByteArray inside `getMockedBlob` function. This will make test code less cluttered which is important when reading tests.